# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../dmraid/destdir.patch
# Copyright (C) 2007 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./make.tmpl.in.orig	2007-06-12 20:01:27.000000000 -0400
+++ ./make.tmpl.in	2007-06-12 20:02:03.000000000 -0400
@@ -25,11 +25,11 @@
 exec_prefix = $(DESTDIR)@exec_prefix@
 bindir = @bindir@
 ifeq ("@KLIBC@", "no")
-  libdir = @libdir@
-  sbindir = @sbindir@
-  includedir = @includedir@
-  infodir = @infodir@
-  mandir = @mandir@
+  libdir = $(DESTDIR)@libdir@
+  sbindir = $(DESTDIR)@sbindir@
+  includedir = $(DESTDIR)@includedir@
+  infodir = $(DESTDIR)@infodir@
+  mandir = $(DESTDIR)@mandir@
 else
   libdir = $(DESTDIR)@KLIBC_LIB_DIR@
   sbindir = $(DESTDIR)@KLIBC_SBIN_DIR@
